Mention istioctl authn tls-check only check conflict states between c… (#4519)
istio/istio.io: content/docs/tasks/security/mutual-tls/index.md

Use the `istioctl` tool to check if the mutual TLS settings are in effect. The `istioctl` command needs the client's pod because the destination rule depends on the client's namespace.
You can also provide the destination service to filter the status to that service only.

{{< tip >}}

> This tool only check the TLS setting consistency between destination rules and authentication policies. It does not take into account whether the

Reviewer: ```suggestion
This tool only checks the consistency of the static TLS settings between destination rules and authentication policies. It doesn't take into account whether or not the
```

> workloads have sidecar or not (i.e, whether those policy/destination rule can be enforced). In other words, status `CONFLICT` doesn't always mean traffic is broken.

Reviewer: ```suggestion
corresponding workloads have sidecars or not. When they don't, the policy and destination rules are not enforced, so note that status `CONFLICT` doesn't always mean that traffic is broken.
```
